core/core_bind.cpp

```diff
@@ -1478,6 +1478,23 @@ TypedArray<Dictionary> ClassDB::class_get_method_list(const StringName &p_class,
 	return ret;
 }
 
+Variant ClassDB::class_call_static_method(const Variant **p_arguments, int p_argcount, Callable::CallError &r_call_error) {
+	if (p_argcount < 2) {
+		r_call_error.error = Callable::CallError::CALL_ERROR_TOO_FEW_ARGUMENTS;
+		return Variant::NIL;
+	}
+	if (!p_arguments[0]->is_string() || !p_arguments[1]->is_string()) {
+		r_call_error.error = Callable::CallError::CALL_ERROR_INVALID_ARGUMENT;
+		return Variant::NIL;
+	}
+	StringName class_ = p_arguments[0]->stringify();
+	StringName method = p_arguments[1]->stringify();
```

Comment:
Why the use of `stringify` here? It is always either a `String` or `StringName`, this makes it look up a `StringName` again if fed one, losing performance

Reply:
Is it implicitly converted? I have no idea how to convert a variant to StringName

Reply:
It is cast, see:
https://github.com/godotengine/godot/blob/71699e08c9df78b7203fa4ef9cede28e995d6ace/core/object/object.cpp#L1138-L1147

For example